src/z3c/recipe/sphinxdoc/__init__.py

```diff
@@ -168,9 +171,11 @@ def install(self):
     update = install
 
 
-def main(projects):
+def main(projects, argv=None, exit_on_error=False):
```

Comment:
I don't *think* so? This recipe is used for generating the docs for many projects at once. I don't think you want it to bail after generating doc number 3 of 50 just because doc number 3 happened to have an error.

It didn't behave that way in the past, AFAICS.